ddev/src/ddev/cli/size/utils/common_funcs.py

```diff
@@ -208,6 +217,22 @@ def get_files(repo_path: str | Path, compressed: bool) -> list[FileDataEntry]:
     ]
 
 
+def check_python_version(repo_path: str, integration_name: str, py_major_version: str) -> bool:
+    pyproject_path = os.path.join(repo_path, integration_name, "pyproject.toml")
+    if os.path.exists(pyproject_path):
+        pyproject = load_toml_file(pyproject_path)
+        if "project" not in pyproject or "classifiers" not in pyproject["project"]:
+            return False
+        classifiers = pyproject["project"]["classifiers"]
+        integration_py_version = ""
+        for classifier in classifiers:
+            match = re.match(r"Programming Language :: Python :: (\d+)", classifier)
+            if match:
+                integration_py_version = match.group(1)
+                return integration_py_version == py_major_version
```

Comment:
```suggestion
        integration_py_version = ""
        pattern = re.compile(r"Programming Language :: Python :: (\d+)")
        for classifier in classifiers:
            match = pattern.match(classifier(
            if match:
                integration_py_version = match.group(1)
                return integration_py_version == py_major_version
```
suggestion: Compiling the regex once and using it to multiple times is more efficient performance wise.

Reply:
Changed! I also changed the other of the regex in the code